great_expectations/dataset/dataset.py

```diff
@@ -4198,6 +4222,76 @@ def expect_column_kl_divergence_to_be_less_than(
 
         return return_obj
 
+    @MetaDataset.column_aggregate_expectation
+    def expect_column_pair_cramers_phi_value_to_be_less_than(
+        self,
+        column_A,
+        column_B,
+        bins_A=None,
+        bins_B=None,
+        n_bins_A=None,
+        n_bins_B=None,
+        threshold=0.05,
+        ignore_missings=None,
+        result_format=None,
+        include_config=True,
+        catch_exceptions=None,
+        meta=None,
+    ):
+        """
+        Expect the values in column_A to be independent of those in column_B.
+
+        Args:
+            column_A (str): The first column name
+            column_B (str): The second column name
+            threshold (float): Maximum allowed value of cramers V for expectation to pass.
+
+        Keyword Args:
+            bins_A (list of float): Bins for column_A if numeric.
+            bins_B (list of float): Bins for column_B if numeric.
+            n_bins_A (int): Number of bins for column_A if numeric. Ignored if bins_A is not None.
+            n_bins_B (int): Number of bins for column_B if numeric. Ignored if bins_B is not None.
+            ignore_missings (bool): \
+                If True, ignore columns where either column has missing. If False, treat missing values as
+                another category (only for methods ...).
```

Comment:
I don't see this kwarg being used, and the decorator's new behavior of reporting not null as BOTH column A and column B not null makes me think perhaps this kwarg should just be removed?